bin/inference/pycbc_inference

```diff
@@ -201,6 +219,37 @@ with ctx:
             ', '.join(sampling_parameters), ', '.join(replace_parameters)))
     else:
         sampling_parameters = replace_parameters = sampling_transforms = None
+    
+    # read in calibration transfer functions
+    if opts.calib and opts.transfer_functions:
+        logging.info("Calibration parameters will be included in likelihood "
+                     "calculation")
+        logging.info("Reading calibration data file")
+        with h5py.File(opts.transfer_functions, "r") as fp:
```

Comment:
This should be a function somewhere, in the ``io`` subpackage or a function of ``Recalibrate``.

Also the parameters should be able to be MCMC'd not fixed like this. We should be able to do something in the configuration file like:
```
[variable_args]
h1-fc0 =

[prior-h1-fc0]
name = gaussian
etc.
```
or
```
[static_args]
h1-fc0 = 123
```

We should treat them as any other variable. And if there is no ``[calibration]`` section but these arguments are supplied, there should be an error raised probably.